app/scripts/modules/core/instance/instanceListBody.directive.js

```diff
@@ -103,17 +103,19 @@ module.exports = angular.module('spinnaker.core.instance.instanceListBody.direct
           switch(filterType) {
             case 'id':
               return a.id.localeCompare(b.id);
-            case 'launchTime':
+            case 'launchTime': {
```

Comment:
I've never seen it either, but yes, the updated linter was complaining.  Here's the writeup:  http://eslint.org/docs/rules/no-case-declarations.  At first, I thought it was the `return` statements.  We could just revert this and turn off the warning.